cmd/oceancenter/dl/hydrophone/pkg/v1.0.2/pkg.json

```diff
@@ -0,0 +1,22 @@
+{
+  "device": "hydrophone",
+  "description": "Install hydrophone release. Must use raspios bullseye or later.",
```

Comment:
This description doesn't need the "Must use raspios bullseye or later" anymore. Now it's just that the image needs to have the microphone drivers installed before the package can be installed, which happens when we prepare an image.